src/components/GDPRConsent.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+
+import { GTM_ID } from '../constants';
+import CookieConsent, { getCookieConsentValue }  from "react-cookie-consent";
+import TagManager from 'react-gtm-module';
+
+
+const GDPRConsent = () => {
+  
+  const onAccept = () => {
+    // Nothing yet
+  };
+  
+  useEffect(() => {
+    // Just to ensure that when we initialize GTM, this message will be shown
```

Comment:
Which message?

Reply:
The message on this component. I improved the comment.